components/zcash_protocol/src/consensus.rs

```diff
@@ -709,12 +731,18 @@ impl BranchId {
                 .activation_height(NetworkUpgrade::Nu5)
```

Comment:
(Not specific to this PR, non-blocking.)

> The return type of this value is slightly more precise than [`Self::height_range`]:

How is it more precise? The return value of `height_range` doesn't appear to lose any information relative to `height_bounds`.

Reply:
This comment was added in 3dc05a69eb691eaaf401813c7a35032b1e4317e9 and I honestly have no recollection of what I meant there. I have a vague recollection that `height_range`, while being in the same commit, may have been modified from an earlier version that was less precise once I figured out that tuples implement `RangeBounds`, and maybe the comment was not updated to reflect that change.